.gitlab/binary_build/linux.yml

```diff
@@ -93,3 +93,35 @@ build_iot_agent-binary_arm64:
     - source /root/.bashrc
     - inv check-go-version
     - inv -e agent.build --flavor iot --major-version 7
+
+build_agentless_scanner-binary_x64:
+  stage: binary_build
+  rules:
+    !reference [.on_a7]
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  needs: ["lint_linux-x64", "tests_deb-x64-py3", "go_deps"]
```

Comment:
Are the `lint_linux-x64` and `tests_deb-x64-py3` jobs related to this one?

Reply:
I'm not sure about `tests_deb-x64-py3`, but the other rules to this, so...

Reply:
I've removed `tests_deb-x64-py3`.